aws/resource_aws_waf_byte_match_set.go

```diff
@@ -24,7 +24,7 @@ func resourceAwsWafByteMatchSet() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
-			"byte_match_tuples": &schema.Schema{
+			"byte_match_tuple": &schema.Schema{
```

Comment:
The documentation in `website/docs/r/waf_byte_match_set.html.markdown` should add the new attribute and add a **deprecated** notice to the existing attribute.